### ISSUE TITLE
L5 suport

### ISSUE THREAD

**Issue by ghost** [michaelkmartin/laravel-magento-integration#1]
Will you suport Laravel 5?

**Comment by Xantios** [comment]
Second this. I'd sure would like to see L5 support.
Although I can imagine its kind of a load to build this to an new version of Laravel.

**Comment by MarketSouth** [comment]
Would love to see L5 support

**Comment by AaronKaa** [comment]
Yeah - come on!! L5

**Comment by Sejido** [comment]
does it work with laravel 5.2 ??

**Comment by Xantios** [comment]
The last time this project was updates is over 2 years ago. hardly can imagine anyone is going to work on this except when somebody forks this ( which im not gonna ;-) )

**Comment by MarketSouth** [comment]
Just use another package like https://github.com/smalot/magento-client.

Im successfully using this package with Laravel 5.1 their are no facades or anything but still works nicely.

Cheers

**Comment by Sejido** [comment]
actually i have an application with laravel 5.2 and i'm trying to use function in mage.php  which is getVersion() i don't know how to do that any suggessesion?

**Comment by Sejido** [comment]
@MarketSouth  thank you very much this tool was so great for me :)

**Comment by Sejido** [comment]
@MarketSouth  thank you so much ,this tool was so useful  ;)

**Comment by mpixelz** [comment]
hi,
is there any chance for this to get an update anytime soon?

**Comment by michaelkmartin** [comment]
Yeah, I've definitely let this project slip a bit, to say the least, sorry about that. This was originally intended to work for Magento 1.x and Laravel 4.x, only. 

My focus, currently, is finishing up a standalone component for both M1 and M2, with providers for Laravel 4 and 5 following soon after.

I'll try to address any bugs with this package. If anyone has a pull request that they'd like to submit, feel free.

**Comment by mpixelz** [comment]
Thanks for the reply @michaelkmartin .. so are we going to get the support for l5? and what do you think when will you be able to do it ?
there is no other git on this and this is the only solution i could find anywhere.

**Comment by michaelkmartin** [comment]
Here's my plan. 

- [ ] Laravel 5 release with SOAP support for M1 and M2. Initial release by **20 March, 2017**
- [ ] Stable release for the Laravel 4 integration with added M2 support. By **3 April, 2017**
- [ ] REST and OAuth support for both integrations. Release TBD

I'm going to break the L5 integration into a separate package, rather than using tags to separate both on this repository. I never actually released a stable version for this repository so it may be less confusing to keep them separate. 

I'll keep this thread updated throughout the L5 upgrade.

**Comment by mpixelz** [comment]
today is the day 😄 
fingers crossed 🤞

**Comment by michaelkmartin** [comment]
Still writing documentation for this but the Laravel5 integration is here.

https://github.com/michaelkmartin/Laravel5-Magento

**Comment by mpixelz** [comment]
@michaelkmartin thanks man.. but im not able to install the package.. it says class not found..

**Comment by Ahmdrza** [comment]
@michaelkmartin can i know if you are done with documentation ? I wanna use your package in my project..

**Comment by Ahmdrza** [comment]
@MarketSouth hi can you please guide me how to use that package?
